classes/class-uagb-helper.php

```diff
@@ -1321,25 +1321,21 @@ public static function get_browser_name( $user_agent ) {
 		 * Get block dynamic CSS selector with filters applied.
 		 *
 		 * @param string $block_name Block name to filter.
-		 * @param array  $selectors Desktop Selectors to filter.
-		 * @param array  $t_selectors Tablet Selectors to filter.
-		 * @param array  $m_selectors Mobile Selectors to filter.
+		 * @param array  $selectors Array of selectors to filter.
 		 * @param array  $attr Attributes.
 		 * @return array Combined selectors array.
 		 */
-		public static function get_combined_selectors( $block_name, $selectors, $t_selectors, $m_selectors, $attr ) {
-			$suffixes           = array( '', '_tablet', '_mobile' );
-			$selector_vars      = array( 
-				'desktop_selectors' => $selectors,
-				'tablet_selectors'  => $t_selectors,
-				'mobile_selectors'  => $m_selectors,
-			);
+		public static function get_combined_selectors( $block_name, $selectors, $attr ) {
 			$combined_selectors = array();
 
-			foreach ( $suffixes as $suffix ) {
-				$key                        = ( '' === $suffix ) ? 'desktop' : ( ( '_tablet' === $suffix ) ? 'tablet' : 'mobile' );
+			if ( ! is_array( $selectors ) ) { 
+				return $combined_selectors;
+			}
+
+			foreach ( $selectors as $key => $selector ) {
+				$hook_prefix                = ( 'desktop' === $key ) ? '' : ( ( '_tablet' === $key ) ? 'tablet' : 'mobile' );
```

Comment:
'_tablet' is that correct?
Please test this code once.

Reply:
```suggestion
				$hook_prefix                = ( 'desktop' === $key ) ? '' : $key;
```

please check if we can use this

Reply:
Updated check